forge-gui/res/cardsfolder/d/dwarven_sea_clan.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:2 R
 Types:Creature Dwarf
 PT:1/1
 A:AB$ Pump | Cost$ T | IsCurse$ True | ValidTgts$ Creature.ControllerControlsIsland+attacking,Creature.ControllerControlsIsland+blocking | TgtPrompt$ Select target attacking or blocking creature whose controller controls an Island | ActivationPhases$ BeginCombat->Combat Damage | ActivationFirstCombat$ True | ConditionDefined$ Targeted | ConditionPresent$ Creature.ControllerControlsIsland | ConditionCompare$ GE1 | SubAbility$ DBDelayTrig | StackDescription$ SpellDescription | SpellDescription$ Choose target attacking or blocking creature whose controller controls an Island. CARDNAME deals 2 damage to that creature at end of combat. Activate only before the end of combat step.
-SVar:DBDelayTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ EndCombat | Execute$ TrigDmg | SpellDescription$ CARDNAME deals 2 damage to that creature at end of combat. | RememberObjects$ ParentTarget | SubAbility$ DBCleanup
+SVar:DBDelayTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ EndCombat | Execute$ TrigDmg | TriggerDescription$ CARDNAME deals 2 damage to that creature at end of combat. | RememberObjects$ ParentTarget | SubAbility$ DBCleanup
```

Comment:
If I understand correctly this should remain SpellDescription ?

Reply:
The card had it twice but can't be cleaned up better imo because of the "Activate only before the end of combat step." that follows after